gcovr/gcov_parser.py

```diff
@@ -332,6 +334,7 @@ def parse_coverage(
             lines=src_lines,
             warnings=_ExclusionRangeWarnings(logger, filename),
             exclude_lines_by_pattern=exclude_lines_by_pattern,
+            exclude_pattern_prefix=exclude_pattern_prefix if exclude_pattern_prefix is not None else r"[GL]COVR?"
```

Comment:
Always use the argument.
```suggestion
            exclude_pattern_prefix=exclude_pattern_prefix,
```

Reply:
Pushed.